File: libraryWallrCustoms/src/main/java/com/zebrostudio/wallrcustoms/customtextview/WallrCustomTextView.java
package com.zebrostudio.wallrcustoms.customtextview;

import android.content.Context;
import android.graphics.Color;
import android.support.v7.widget.AppCompatTextView;
import android.util.AttributeSet;

public class WallrCustomTextView extends AppCompatTextView {

  public WallrCustomTextView(Context context) {
    this(context, null);
  }

  public WallrCustomTextView(Context context, AttributeSet attrs){
    this(context, attrs,1);
  }

  public WallrCustomTextView(Context context, AttributeSet attrs, int defStyleAttr) {
    super(context, attrs, defStyleAttr);
    this.setTextColor(Color.WHITE);
    this.setTypeface(FontCache.getRobotoRegularTypeface(context));
  }
}

File: libraryWallrCustoms/src/main/java/com/zebrostudio/wallrcustoms/customtextview/WallrCustomTextView.kt
package com.zebrostudio.wallrcustoms.customtextview

import android.content.Context
import android.graphics.Color
import androidx.appcompat.widget.AppCompatTextView
import android.util.AttributeSet

class WallrCustomTextView @JvmOverloads constructor(
  context: Context,
  attrs: AttributeSet? = null,
  defStyleAttr: Int = 1
) : AppCompatTextView(context, attrs, defStyleAttr) {

  init {
    this.setTextColor(Color.WHITE)
    this.typeface = FontCache.getRobotoRegularTypeface(context)
  }
}
